model/finished_stages.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import "time"
+
+type FinishedStages struct {
+	Stages []Stage
+}
+
+type Stage struct {
+	ID        string     `json:"id"`
+	Name      string     `json:"name"`
+	Workflows []Workflow `json:"workflows"`
+}
+
+type Workflow struct {
```

Comment:
Should we or should we not explicitly define all fields here coming from the configuration? Some fields are not used from this struct in the step.